INSTALL.rst

```diff
@@ -45,12 +45,13 @@ of all dependencies we recommend using `homebrew <https://brew.sh/>`__:
 
    brew install flex bison cmake python3
 
-The necessary Python packages can then easily be added using the pip3
-command.
+All of the Python dependencies (build, run, and development) can be installed
+using:
 
 .. code:: sh
 
-   pip3 install --user -r requirements.txt
+   pip3 install --user pip-tools
+   pip3 install --user -r <(pip-compile --all-build-deps --all-extras --no-strip-extras 2>&1)
```

Comment:
The question is if it's worth duplicating some information to avoid these instructions and instead have the standard, easily discoverable:
```
pip install --user -r requirements.txt
```

Reply:
Yeah it's a bit annoying, I've put back requirements now.